presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/PushDownNegationsExpressionRewriter.java

```diff
@@ -81,9 +81,9 @@ public Expression rewriteNotExpression(NotExpression node, Void context, Express
                 checkState(leftType != null && rightType != null, "missing type for expression");
                 if ((typeHasNaN(leftType) || typeHasNaN(rightType)) && (
                         operator == GREATER_THAN_OR_EQUAL ||
-                        operator == GREATER_THAN ||
-                        operator == LESS_THAN_OR_EQUAL ||
-                        operator == LESS_THAN)) {
+                                operator == GREATER_THAN ||
+                                operator == LESS_THAN_OR_EQUAL ||
+                                operator == LESS_THAN)) {
```

Comment:
This is less readable than before

Reply:
Same as above. Another option is to do
```java
if ((typeHasNaN(leftType) || typeHasNaN(rightType))
        && (operator == GREATER_THAN_OR_EQUAL ||
        operator == GREATER_THAN ||
        operator == LESS_THAN_OR_EQUAL ||
        operator == LESS_THAN)) {
```
Or this could use
```java
if ((typeHasNaN(leftType) || typeHasNaN(rightType)) && EnumSet.of(
        GREATER_THAN_OR_EQUAL,
        GREATER_THAN,
        LESS_THAN_OR_EQUAL,
        LESS_THAN)
        .contains(operator)) {
```
But those don't look like an improvement.